docs/source/exporters/tflite/overview.mdx

```diff
@@ -0,0 +1,35 @@
+ 
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Overview
+
+🤗 Optimum handles the export of PyTorch or TensorFlow models to TFLite in the `exporters.tflite` module. It provides classes, functions, and a command line interface to perform the export easily.
```

Comment:
```suggestion
🤗 Optimum handles the export of TensorFlow models to TFLite in the `exporters.tflite` module. It provides classes, functions, and a command line interface to perform the export easily.
```
Maybe stress on the fact that we can load PyTorch weights in a TensorFlow model.

Reply:
done